www/main/toolbar.js

```diff
@@ -153,12 +153,19 @@ const setListeners = () => {
 };
 
 const remoteSyncInit = async () => {
+  const $syncConent = document.querySelector('.sync-content-wrapper');
+  $syncConent.classList.add('loading');
   const onlineVal = await isOnline();
   if (isRequestSent) {
     return;
   }
   if (onlineVal) {
     const newCode = await tryConnection();
+    if (newCode) {
+      $syncConent.classList.remove('loading');
+    } else {
+      remoteSyncInit();
```

Comment:
this could get stuck in an endless loop if the service is down. We should handle the error better in tryConnection.